components/tyk-bootstrap/values.yaml

```diff
@@ -68,6 +68,9 @@ bootstrap:
   # Trigger to control if we want to create the tyk-operator secret
   operatorSecret: tyk-operator-conf
 
+  # The default name for the sa is k8s-bootstrap-role
```

Comment:
Please also add a comment what happens if users configure this field. For instance:
```
serviceAccountName field configures the name of the ServiceAccount that is going to be created for bootstrapping Jobs
```